components/definitions/definitions.js

```diff
@@ -0,0 +1,158 @@
+export const classDialogDocument = 'popup__document';
+
+export const idDialogMessage = 'd-message';
+
+export const attrResponse = 'data-popup-response';
+export const attrDataDialogCall = 'data-popup-call';
+export const attrDataTitle = 'data-title';
+
+export default function() {
+  return definitionDialog();
+}
+
+export function definitionDialog() {
+  anchorsToButton();
+  buttonClick();
+}
+
+// Replace anchor links with buttons
+export function anchorsToButton() {
+  let anchors = document.getElementsByClassName(classAnchor);
+  for (let anchor of anchors) {
+    // Replace <a> tag with <button> tag.
+    anchor.outerHTML = anchor.outerHTML
+      .replace(/<a/g, '<button')
+      .replace(/<\/a>/g, '</button>');
+  }
+}
+
+// Open popup
+export function buttonClick() {
+  const buttons = document.getElementsByClassName(classAnchor);
+  for (let button of buttons) {
+    button.onclick = function(e) {
+      e.preventDefault();
+
+      // define the popup element
+      let popup = document.getElementsByClassName(classDialog)[0];
+
+      // record the trigger element
+      let trigger = button.getAttribute('id')
+        ? button.getAttribute('id')
+        : 'trigger';
+
+      if (popup.getAttribute('open') === 'true') {
+        closeDialog(buttons, trigger, popup);
+      }
+
+      openDialog(buttons, button, popup, trigger);
+    };
+  }
+}
+
+// Open Dialog
+export function openDialog(buttons, button, popup, trigger) {
+  // retrieve custom close button wording, if any
+  let closeText = button.getAttribute(attrResponse)
+    ? button.getAttribute(attrResponse)
+    : 'close';
+
+  button.setAttribute('disabled', 'true');
+
+  // open popup and add roles
+  popup.setAttribute('tabindex', '0');
+  popup.setAttribute('open', 'true');
+  popup.setAttribute('role', 'alertpopup');
+  popup.setAttribute('aria-labelledby', idDialogMessage);
+
+  // build the popup markup
+  popup.insertAdjacentHTML(
+    'beforeend',
+    '<div class="popup__wrapper">' +
+      '<div class="' + classDialogDocument + '" id="document" role="document" tabindex="0">' +
+        '<button class="' + classDialogButton + ' mars" id="button" role="button">' +
+          closeText +
+        '</button>' +
+      '</div>' +
+    '</div>'
+  );
+
+  // Define content to refocus popup if user tries to leave it
+  let content = document.getElementsByClassName(classDialogDocument)[0];
+
+  // make last button in popup the close button
+  let btnClose = document.getElementsByClassName(classDialogButton)[0];
+  btnClose.focus();
+
+  // Insert the message held in the trigger's [data-popup-msg] attribute
+  let dataDialogCall = button.getAttribute(attrDataDialogCall);
+  let dataTitle = button.getAttribute(attrDataTitle);
+
+  btnClose.insertAdjacentHTML(
+    'beforebegin',
+    '<div id="' + idDialogMessage + '" class="popup__message" ><h3 class="popup__title venus">' + dataTitle + '</h3><p class="popup__description mars" >' + dataDialogCall + '</p></div>'
```

Comment:
Use a template literal (as above) with classes declared as `const` to make more readable.